lib/common/zstd_internal.h

```diff
@@ -299,6 +299,7 @@ typedef struct {
     BYTE*  ofCode;
     size_t maxNbSeq;
     size_t maxNbLit;
+    BYTE* longOffsets;
```

Comment:
This doesn't need to be a pointer.

```suggestion
    int longOffsets;
```

Reply:
There are a bunch of places I have to un-const if I want to change this (I think).

Reply:
Makes sense, then instead of storing it in the `seqStore`, we can have `ZSTD_seqToCodes()` return that value, and store it in the `ZSTD_symbolEncodingTypeStats_t` returned by `ZSTD_buildSequencesStatistics()`.